drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -70,6 +70,7 @@ def definition
 
     # EXPENSIVE! Do not use in batch
     def upgraded_definition_for_editing
+      return if object.in_progress? # WIP assessments should use the original form for editing
```

Comment:
This is a change in behavior. The frontend was previously implementing this logic, but it is simpler for the backend to handle it.